### PR TITLE
Summary of the conventions

### PR DESCRIPTION
**Pull request by vEnhance**
WIP, this is like half done, but throwing out a draft now to get impressions from people.

In short, this is supposed to be a high-level overview of the commonly used conventions in basic levels 1-25.

Similar to #959/#990 but organized conceptually rather than directly by level (and we have a script to shuffle levels now), and only covers a subset of the basic conventions (and not the extras). So I think it should not be too much maintenance work (and I can certainly help with that).

### CODE REVIEW

**Comment by vEnhance** [comment]
OK done writing the cheat sheet!

**Comment by Zamiell** [comment]
can you rebase on main?

**Comment by vEnhance** [comment]
> can you rebase on main?
done

(also, in `learning-path.mdx`, the big diff is just because i added the word "level" in front of the number, none of the lists got edited)

**Comment by Zamiell** [comment]
yeah thats why i updated main

**Comment by Zamiell** [comment]
i can see the diff more clearly now

File: docs/learning-path.mdx [hunk 1]
```diff
@@ -24,6 +24,7 @@ For information on how variants tie in to the levels, see the [variant summary](
 ### Level Summary
 
 The following is a quick summary of the levels.
+You can also see a [cheat sheet of many of the main conventions](summary.mdx).
```
Comment (Zamiell) [inline docs/learning-path.mdx:27]: this won't appear on a newline, is that intended? if not, then put it on the same line

**Pull request merged**